config.tfvars

```diff
@@ -243,7 +243,7 @@ bamboo_local_home_size = "10Gi"
 # Number of Bamboo remote agents to launch
 # To install and use the Bamboo agents, you need to provide pre-seed data including a valid Bamboo license and system admin information.
 # Feel free to uncomment the following line and supply the number of remote agents.
```

Comment:
This comment probably isnt needed anymore now that the value is not commented out.